src/core/sagas/webui.ts

```diff
@@ -14,7 +14,7 @@ import { setItem } from '../slices/misc';
 
 function* checkUpdates() {
   yield put(startFetching('checkingUpdates'));
-  const channel = yield select((state: RootState) => state.webuiSettings.v3.updateChannel);
+  const channel = yield select((state: RootState) => state.webuiSettings.webui_v2.updateChannel);
```

Comment:
do we really want to keep v1 and v2 in same key? Maybe add some parameter to settings for version and server could use totally separate clean key or it ?